root/static/styles/favicons.less

```diff
@@ -34,6 +34,7 @@
 .favicon("genius");
 .favicon("googleplus", 32);
 .favicon("ibdb");
+.favicon("IMDb");
```

Comment:
I only looked at the ones in the diff. :) I would go with the more prevalent style, which, based on the diffs, seem to be all lower vase.